python/ql/src/experimental/Security/CWE-208/TimingAttack.qhelp

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+  Timing Attack is based on the leakage of information of secret parameters by studying
+how long it takes the system to respond to different inputs. 
+  it can be circumvented by using a constant-time algorithm for checking the value of sensitive info, 
+more precisely, the comparison time should not depend on the content of the input. Otherwise the attacker gains
+information that is indirectly leaked by the application. This information is then used for malicious purposes, 
+such as guessing the password of a user.
```

Comment:
Do you have any information that suggests that such attacks (guessing the password of a user) are
1. done in practice
2. done over the internet
?

Reply:
Hi, @intrigus-lgtm 
I never heard.